common/browser.go

```diff
@@ -338,7 +334,6 @@ func (b *Browser) isAttachedPageValid(ev *target.EventAttachedToTarget, browserC
 		return false
 	}
 	// If the target is not in the same browser context as the current one, ignore it.
-	// We're not adding a log to prevent overhead.
 	if browserCtx.id != targetPage.BrowserContextID {
 		b.logger.Warnf(
```

Comment:
Should this be a debug? It sounds like we're going to get a lot of the logs when running multi VU tests which is only going to add to the noise and confuse users.

Reply:
Done: c8416b3caf2bf1add6359d78f666c343a1c1124e